bool request

-----
float32[] x
float32[] y
float32[] theta
float32[] vb
float32[] phi
float32[] xt
float32[] yt
bool[] moveable
bool[] crash
bool[] reach